list/lib/listitemlink/list-item-link-only.ts

```diff
@@ -27,4 +27,8 @@ export class ListItemLinkOnly extends ListItemLink {
   override renderRipple() {
     return this.noninteractive ? nothing : super.renderRipple();
   }
+
+  override renderFocusRing() {
```

Comment:
With https://github.com/material-components/material-web/pull/4337 in flight, can we limit the scope of this PR to just adding `{type: Boolean}`?

Reply:
@asyncLiz not same file :eyes:

Reply:
@asyncLiz I can if you want remove `renderFocusRing` if you prefer having this change in #4333 but can @VandeurenGlenn really do it without messing things is the question :eyes:

Reply:
@vdegenne I was doing to much at once ok 😂